site/topics/control-logic/instructions-microcodes.rst

```diff
@@ -126,8 +126,12 @@ Microcodes
 
 * Below is a table showing how the control lines would be configured for the two steps
 
-    * The data and clock columns are excluded
     * Like before, each row corresponds to one clock cycle
+    * Due to space limitations, the data and clock columns are removed some control signals' columns are combined
+
+        * :math:`ALU_{o}` and :math:`sub` are combined (``alu/sub``)
+        * :math:`Out_{i}` and :math:`sign` are combined (``output/sign``)
+        * :math:`PC` and :math:`PC_{e}` are combined (``in/out/enable``)
 
 
 .. list-table:: Control logic for loading data from some memory address to register A
```

Comment:
I shrunk this table 

![image](https://github.com/user-attachments/assets/05e06eda-0e03-4eb4-b5de-14234005301c)